example/.mage/magefile.go

```diff
@@ -5,7 +5,7 @@ package main
 
 import (
 	"github.com/magefile/mage/mg"
-	"go.einride.tech/mage-tools/mgmake"
+	"go.einride.tech/mage-tools/mgbuild/mgmake"
```

Comment:
does `mgmake` need to be a child, or can we maintain the flat structure by putting `mgmake` at the same level?

Reply:
It does not

Reply:
Disregard me - since it's a command let's go with the comment below!